examples/with-client/ponder/src/api/index.ts

```diff
@@ -1,7 +1,7 @@
 import { type ApiContext, ponder } from "@/generated";
 import { trpcServer } from "@hono/trpc-server";
-import { eq } from "@ponder/core";
 import { initTRPC } from "@trpc/server";
+import { eq } from "ponder";
```

Comment:
Is there any value to using different entrypoints here? 
```ts
import { eq } from "ponder/db";
// or 
import { eq } from "ponder/drizzle";
// or 
import { eq } from "ponder/sql";
```
Everything from top-level seems fine/simple, just curious if you gave it any thought.

Reply:
Let's take another look on monday